### ISSUE TITLE
GMail giving Certificate Error (IMAP)

### ISSUE THREAD

**Issue by jnylen** [Foundry376/Mailspring#62]
##### Are there any related issues?
Not that I know of

##### What operating system are you using?
Archlinux using aur that builds from the .deb

##### What version of Mailspring are you using?
1.0.2

--

**Bug?**
##### Do you have any third-party plugins installed? If so, which ones?
No

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?
Gmail

##### Is the issue reproducible with a particular attachment, message, signature, etc?
Basically it works if I disable SSL Verifcation in the IMAP setup rather than in the gmail one.

**Comment by domsle** [comment]
I am having exactly the same issue.

**Comment by ptkdev** [comment]
+1

**Comment by Lerentis** [comment]
Having the same problem with any email server. the log only says this
```
----------IMAP----------
* OK [CAPABILITY IMAP4rev1 LITERAL+ SASL-IR LOGIN-REFERRALS ID ENABLE IDLE AUTH=PLAIN AUTH=LOGIN] Dovecot ready.
```

**Comment by bengotow** [comment]
Hey folks—I /think/ this is related to https://github.com/Foundry376/Mailspring/issues/63. We may want to ship that first and see if this issue is resolved.

**Comment by jnylen** [comment]
Fixed by 1.0.3

**Comment by kostaskol** [comment]
Not sure if I should comment here or open a new issue entirely. At any rate, it seems that this issue is back in 1.0.5-1 on Arch Linux when adding custom IMAP/SMTP account.

**Comment by bengotow** [comment]
Hey! Can you confirm that this happens for you in 1.0.5 with a mail
provider you know should be trusted, like yahoo or gmail? In some cases
this error can be valid and correct, if your imap/smtp server is using an
expired certificate or a self signed one.

If it’s only happening with a particular server you can disable the check
by checking the “allow insecure ssl” checkbox. Then it should work fine.

Ben

**Comment by kostaskol** [comment]
You're absolutely right. The certificate had indeed expired. I'll be sure to let them know. Feel free to delete my comments and sorry for commenting on a closed thread. 
Best wishes :)

**Comment by riki137** [comment]
Still happening to me. Can't use the app. Latest Windows 10, November 2018.

**Comment by MrTruth0** [comment]
Same as Riki ..

**Comment by CodeMouse92** [comment]
It appears that there has been no activity on this issue for some time, so we will close it. **If the problem occurs again, please create a new thread in the appropriate category on our [Discourse](https://community.getmailspring.com).**

We are in the process of migrating issues to Discourse, which can better facilitate discussion and discovery, and so GitHub Issues can focus on issues that are confirmed and slated for resolution in the near term. [Learn more about the changes here.](https://community.getmailspring.com/t/mailspring-community-transition/38)

We hope to see you there soon!

-The Mailspring Team
